app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -126,6 +126,10 @@ def attention_with_barcode?
             parsed_alerts.dig(:failed, 0, :result) == 'Attention'
         end
 
+        def billed?
+          !!doc_auth_result
+        end
```

Comment:
I moved this up because it was private before and now it's public so I could test it.

I also could have just called `.send()` in the spec but 🤷 it seems like something we'd want to expose